portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,40 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
Suggest renaming this something other than "Data" - "Comments" or something, maybe. Up to you